project_event/tests/test_project_event_task.py

```diff
@@ -25,6 +25,70 @@ def setUp(self):
             'date_end': fields.Datetime.to_string(datetime.today() + timedelta(hours=4)),
         })
 
+        self.task_2 = self.Tasks.create({
+            'name': 'Test Task 2',
+            'activity_task_type': 'task',
+            'project_id': self.project_1.id,
+            'responsible_id': self.project_1.responsible_id.id,
+            'partner_id': self.project_1.partner_id.id,
+            'room_id': self.room_1.id,
+            'parent_id': None,
+            'date_start': fields.Datetime.to_string(datetime.today()),
+            'date_end': fields.Datetime.to_string(datetime.today() + timedelta(hours=4)),
+        })
+
+    def test_0071_create_activity(self):
```

Comment:
@eilst why 0071, you think that we can have more than 99 tests for project tasks?

Reply:
@eilst we add test by an interval of 10 in case if we want to add other new tests between test_010 and test 020 for example.